all/embedded/pom.xml

```diff
@@ -325,12 +349,16 @@
                            !org.jgroups.*,
                            !org.jboss.logmanager,
                            !org.jboss.modules,
+                           !org.jboss.dmr,
                            !org.iq80.snappy,
                            !org.jboss.marshalling.*,
                            !org.xerial.snappy,
                            !sun.misc,
                            !sun.reflect,
                            !sun.nio.ch,
+                           !__redirected,
+                           com.sun.net.httpserver;resolution:=optional,
+                           io.undertow.*;resolution:=optional,
```

Comment:
Interesting piece for review: Use "better" http servers (like Undertow) when available.